examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>conc: a concentration parameter that controls the how much the probability vectors for each batch of observations can vary. Low concentration values allow for high variance (over dispersed data), while larger values lead to more similarly distributed data. In the limit of a large concentration, the model approaches the Multinomial distribution. (Not the exact wording, just a suggestion).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<blockquote>a concentration parameter that controls the how much the probability vectors for each batch of observations can vary. Low concentration values allow for high variance (over dispersed data), while larger values lead to more similarly distributed data. In the limit of a large concentration, the model approaches the Multinomial distribution</blockquote><p> </p><p>I feel like I already have most of that, but I could be missing something. Can you explain more about which details in your suggested text are missing or less clear in the current description of the <code>conc</code> parameter:</p><p> </p><blockquote>controlling the overdispersion of our data, where larger values result in our distribution more closely approximating the multinomial. </blockquote><p> </p><p>Is "overdispersion" to lingo-y and you're suggesting that I focus on referring to "variance"?</p>

Reply:
I mostly meant to explicit call it concentration and explain it as such (as some might not get it from the abbreviated conc or the python code) because it is a very common type of parameter in Bayesian modeling and it might help readers connect with previous knowledge